### ISSUE TITLE
Link to issue against upstream Eigen?

### ISSUE THREAD

**Issue by EricCousineau-TRI** [ros2/eigen3_cmake_module#5]
README mentions defect, but shows no traceability towards letting upstream know, nor does it give users of this package a way to know if/when they can stop using it depending on their Eigen version.

\cc @IanTheEngineer @sloretz

**Comment by clalancette** [comment]
This package may not be needed anymore in Ubuntu 20.04 and later.  If that's the case, we would love to retire it, but we'd have to verify that all of the known issues with finding Eigen are gone first.

**Comment by EricCousineau-TRI** [comment]
Thanks! Though it's not immediately clear to me why it may not be needed anymore in future OS versions.

Was there an upstream patch that we should keep an eye out for?

**Comment by EricCousineau-TRI** [comment]
My guess is maybe this one? https://gitlab.com/libeigen/eigen/-/issues/1386

**Comment by clalancette** [comment]
> My guess is maybe this one? https://gitlab.com/libeigen/eigen/-/issues/1386

Yeah, possibly.  I'm going off of my memory here (since we don't have this written down), but my understanding is that later Eigen has fixed the CMake issues that caused this package to be needed in the first place.  If we can verify that on our target platforms, then we can consider removing this.

**Comment by sloretz** [comment]
IIRC the current state is Eigen's upstream's CMake should work great now that ROS 2 uses modern CMake targets, except that we ship our own `Eigen3Config.cmake` in the chocolatey package which doesn't make the exported targets available: https://github.com/ros2/choco-packages/issues/19 . I think this cmake module can be removed once we fix the chocolatey package.

**Comment by RFRIEDM-Trimble** [comment]
Works great for me following this guide on galactic ubuntu 20.04.
https://eigen.tuxfamily.org/dox/TopicCMakeGuide.html

package.xml for my project - use the rosdep source of `eigen`, which pulls in eigen3 on Ubuntu. 
```
 <depend>eigen</build>
```
CMakeLists for my project:
```
find_package (Eigen3 3.3 REQUIRED NO_MODULE)

ament_auto_find_build_dependencies(..)
ament_auto_add_library(..)
target_link_libraries(${PROJECT_NAME}_lib Eigen3::Eigen)
```
